src/lib/grammar.rs

```diff
@@ -69,6 +69,18 @@ pub enum AssocKind {
     Nonassoc
 }
 
+impl Grammar {
```

Comment:
Yep. You can even have them in different modules, to do a sort of "statically typed monkey patching".